### DISCUSSION TITLE
Contributors are not loading since 3 weeks ago

### DISCUSSION THREAD

**Discussion by d3xt3r2909** [community/community#58008] [Repositories]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues (The feature that allows you to track ideas, feedback, tasks, or bugs for work on GitHub. Not for any issue you are having).
- [ ] Projects (For planning and tracking work on GitHub - not any open source or other type of project).

### Body

Hi, 

3 weeks ago contributors list just stop loading with message 

> We tried our best, but the graph wouldn’t load. Try reloading the page.

I wouldn't report this as a bug ofc but if it continuously happens with the same message then I need to assume it is?

<img width="983" alt="image" src="https://github.com/community/community/assets/17637299/623a0c07-f59f-46ca-8e05-1e0450d2c53c">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @d3xt3r2909 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by d3xt3r2909** [comment]
Thank you @queenofcorgis, need to learn more about structure.

**Comment by Deeven-Seru** [comment]
do we have an answer yet ? :)
